### PR TITLE
Update HelloSign 2FA options

### PR DESCRIPTION
**Pull request by jspaetzel**
HelloSign has recently added software 2FA support. More details about how to enable this one the website: https://faq.hellosign.com/hc/en-us/articles/360025164091-Two-Factor-Authentication-Google-Authenticator

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
looks good

File: _data/legal.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ websites:
       img: hellosign.png
       tfa: Yes
       sms: Yes
+      software: Yes
       exceptions:
-          text: "Two-factor authentication is only available to paid Business plan senders and above."
-      doc: https://faq.hellosign.com/hc/en-us/articles/205714548-How-do-I-set-up-two-factor-authentication-
+          text: "SMS Two-factor authentication is only available to paid Business plan senders and above."
+      doc: https://faq.hellosign.com/hc/en-us/articles/360025164091-Two-Factor-Authentication-Google-Authenticator
```
Comment (Carlgo11) [inline _data/legal.yml]: On Zendesk pages, the text that's behind the numbers in the URL aren't actually needed.

```suggestion
      doc: https://faq.hellosign.com/hc/en-us/articles/360025164091
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/legal.yml (hunk 1 above)
Comment (jspaetzel) [reply _data/legal.yml]: Good suggestion, thanks!

**Review by Carlgo11 — APPROVED** [review]

**Review by jtagcat — APPROVED** [review]
Why would they want sms for only business, should rather non-business...

**Review by jamcat22 — APPROVED** [review]

**Comment by jamcat22** [comment]
@vc7ee typically when sites restrict types of 2FA to paid plans, it’s to help offset the cost associated with providing that type of 2FA. For example, it doesn’t directly cost HelloSign any additional money to process Google Authenticator tokens, but sending SMS messages has a direct cost per authentication request. So for free users they provide the option that doesn’t directly cost them anything, while for paid users they offer an additional option that directly costs them money each time someone logs in (likely between $0.01-$0.05 per message + $0-$30 a month depending on service and country code).

**Pull request merged**